### PR TITLE
[v1.10] Added in new Algolia search system

### PR DESCRIPTION
**Pull request by dfalling**
This commit applies the same search changes as in v1.11.0.

### CODE REVIEW

File: data/search.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-engine_key: 'UdruvUp_iEcu2wdYqw9D'
-document_type: 'guides-v1-10-0'
-revision: 'v1.10.0'
+api_key: '760969ef081fcadc7e0e60faefdb0907'
+document_type: 'guides-v1-11-0'
+revision: 'v1.11.0'
```
Comment (acorncom) [inline data/search.yml]: Could you change these back to v1.10.0?

**Comment by acorncom** [comment]
Helps with https://github.com/emberjs/guides/issues/1561

File: data/search.yml (hunk 1 above)
Comment (dfalling) [inline data/search.yml]: Fixed.

**Comment by acorncom** [comment]
CI failures were occurring for guides at this point in time, these issues are not with this PR

**Pull request merged**